tembo-operator/src/app_service/manager.rs

```diff
@@ -692,29 +702,33 @@ pub async fn reconcile_app_services(cdb: &CoreDB, ctx: Arc<Context>) -> Result<(
         }
     }
 
-    match reconcile_ingress_tcp(
-        client.clone(),
-        &coredb_name,
-        &ns,
-        oref.clone(),
-        desired_tcp_routes,
-        desired_middlewares,
-        desired_entry_points_tcp,
-    )
-    .await
-    {
-        Ok(_) => {
-            debug!("Updated/applied IngressRouteTCP for {}.{}", ns, coredb_name,);
-        }
-        Err(e) => {
-            error!(
-                "Failed to update/apply IngressRouteTCP {}.{}: {}",
-                ns, coredb_name, e
-            );
-            has_errors = true;
+    for appsvc in appsvcs.iter() {
```

Comment:
As a follow-up, let's make sure the expected `IngressRouteTCP` resources are created when we have multiple apps that use `IngressType: tcp`